src/Report/ReportInterface.php

```diff
@@ -0,0 +1,54 @@
+<?php
+
+/*
+ * This file is part of PHP CS Fixer.
+ *
+ * (c) Fabien Potencier <fabien@symfony.com>
+ *     Dariusz Rumiński <dariusz.ruminski@gmail.com>
+ *
+ * This source file is subject to the MIT license that is bundled
+ * with this source code in the file LICENSE.
+ */
+
+namespace PhpCsFixer\Report;
+
+use Symfony\Component\Stopwatch\Stopwatch;
+
+interface ReportInterface
+{
+    /**
+     * @param int $showAppliedFixers
+     */
+    public function setShowAppliedFixers($showAppliedFixers);
+
+    /**
+     * @param bool $needDiff
+     */
+    public function setShowDiff($needDiff);
+
+    /**
+     * @param bool $dryRun
+     */
+    public function setDryRun($dryRun);
+
+    /**
+     * @param bool $decoratedOutput
+     */
+    public function setDecoratedOutput($decoratedOutput);
+
+    /**
+     * Sets the Stopwatch.
+     *
+     * @param Stopwatch $stopwatch
+     */
+    public function setStopwatch(Stopwatch $stopwatch);
```

Comment:
@keradus How can I pass dry-run or stopwatch to constructor if I know only about interface when creating report class.
```php
protected function createReport($format, InputInterface $input, OutputInterface $output)
    {
        $reportClass = self::$reports[$format];

        /** @var ReportInterface $report */
        $report = new $reportClass();
        $report->setShowAppliedFixers(OutputInterface::VERBOSITY_VERBOSE <= $output->getVerbosity());
        $report->setDryRun((bool) $input->getOption('dry-run'));
        $report->setStopwatch($this->stopwatch);

        return $report;
    }
```

For stopwatch I can create `StopwatchAwareInterface` and calls setter if report implements it. But what about dry-run or showAppliedFixers?